src/components/PostCard/PostCard.tsx

```diff
@@ -27,7 +30,9 @@ export const PostCard = (prop: Posts) => {
   //   time = moment(createdAt?.toDate()).fromNow();
   // }
 
-  const [openCommentSection, setOpenCommentSection] = useState(false);
+  const { id } = useAppSelector((store) => store.auth);
```

Comment:
Suggestion: can use optional chaining (?) to check that each reference in the chain is valid